server/job/plexsync/index.ts

```diff
@@ -182,6 +231,68 @@ class JobPlexSync {
         if (matchedtmdb?.[1]) {
           tvShow = await this.tmdb.getTvShow({ tvId: Number(matchedtmdb[1]) });
         }
+      } else if (metadata.guid.match(hamaTvdbRegex)) {
+        const matched = metadata.guid.match(hamaTvdbRegex);
+        const tvdbId = matched?.[1];
+
+        if (tvdbId) {
+          tvShow = await this.tmdb.getShowByTvdbId({ tvdbId: Number(tvdbId) });
+          if (animeList.isLoaded()) {
+            await this.processHamaSpecials(plexitem, metadata, Number(tvdbId));
+          } else {
+            this.log(
+              `Hama id ${plexitem.guid} detected, but library agent is not set to Hama`,
+              'info'
```

Comment:
Might be better to have the log level "warn" here? Will need to set it to the accepted log levels in the private log method though.